### DISCUSSION TITLE
Link User(Group) to Specific Customer

### DISCUSSION THREAD

**Discussion by 0xab3d** [dfir-iris/iris-web#257] [Ideas]
Hello,

I am not sure if this is already supported or not, I have been looking into the option but to no avail. It would be possible to link specific groups (inherently users) to specific customers.

For example, grp1 (full access) and grp2 (read only) can see only alerts and cases for customer 1
grp3 (full access) and grp4 (read only) can see only alerts and cases for customer 2
admin-grp (full access) can see all customers.

**Comment by whikernel** [comment]
Hi @0xab3d  

Indeed it was not yet officially proposed, but this is something we have in mind and would like to add. It would make things way easier, notably for MSSPs. 👍

**Reply by 0xab3d** [reply]
Thanks. I will close the discussion then for now.
